admin-dev/themes/new-theme/scss/components/layout/_content_div.scss

```diff
@@ -1,8 +1,6 @@
 .content-div {
   // stylelint-disable
-  padding-top:
-    $size-header-height + $size-header-toolbar-height +
-    ($grid-gutter-width / 2);
+  padding-top: 1rem;
```

Comment:
These changes comes from the header calculation removed PR but it seems that JS changes arent in this PR, is it normal?

Specially speaking about this: 

https://github.com/PrestaShop/PrestaShop/pull/26613/files#diff-d8ca3856019b15293606d299b5a3e45fa4085d4a984643c742100d87aafb8ab1L158-L178

It will break the layout if it's not linked to these SCSS change